scripts/releaseNotes.mjs

```diff
@@ -105,6 +107,52 @@ const releaseNotes = (
 );
 console.error('📦 release notes JSON generated');
 
+const validateReleaseNotesAgents = (json) => {
```

Comment:
maybe a nit but i think we should name this arg `releaseNotes`. `json` might imply that it's already stringified

```suggestion
const validateReleaseNotesAgents = (releaseNotes) => {
```